docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -0,0 +1,281 @@
+# Create a OpenStack cluster
+
+Install an OCP cluster running on VMs within a management OCP cluster
+
+## Limitations
+
+* The HyperShift Operator with OpenStack support is currently in development and is not intended for production use.
+* Ingress and OpenStack CSI (Cinder and Manila) are not functional.
+
+## Prerequisites
+
+* Admin access to an OpenShift cluster (version 4.16+) specified by the `KUBECONFIG` environment variable.
+* The Management OCP cluster must be configured with OVNKubernetes as the default pod network CNI.
+* The OpenShift CLI (`oc`) or Kubernetes CLI (`kubectl`).
+* A valid [pull secret](https://console.redhat.com/openshift/install/platform-agnostic/user-provisioned) file for the `quay.io/openshift-release-dev` repository.
+
+## Installing HyperShift Operator and cli tooling
+
+Before creating a guest cluster, the hcp cli, hypershift cli, and HyperShift
+Operator must be installed.
+
+The `hypershift` cli tool is a development tool that is used to install
+developer builds of the HyperShift Operator.
+
+The `hcp` cli tool is used to manage the creation and destruction of guest
```

Comment:
Why not use the hypershift cli for this?

The hcp cli is the productized version of the hypershift cli.

Reply:
this is literally copied from Kubevirt page.

Reply:
after a second thought, do you prefer that we don't document the `hcp` CLI for OpenStack because it's dev preview? I don't have a strong opinion.

Reply:
Ah yeah I forgot Kubevirt did that. I don't have a strong opinion either way.